### DISCUSSION TITLE
Add "dropdown" to select dynamic variables in user role permissions

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#11930] [Feature Requests]
Right now it is possible to create rules for user roles, that depend on dynamic variables like $CURRENT_USER.id.

But this is very hard to set up from the user interface and you have to know those system values that are available.

I think it would be very nice to have some sort of dropdown, for the right hand side of those `rules` so that you can just select something like $CURRENT_USER.id from the gui without using confusing code constructs like it is done right now.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
